[2024-12-29T16:34:13.936+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-29T16:34:13.969+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gold_finance_data_pipeline.pyspark_task manual__2024-12-29T16:34:08.220903+00:00 [queued]>
[2024-12-29T16:34:13.983+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gold_finance_data_pipeline.pyspark_task manual__2024-12-29T16:34:08.220903+00:00 [queued]>
[2024-12-29T16:34:13.983+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 3
[2024-12-29T16:34:14.007+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): pyspark_task> on 2024-12-29 16:34:08.220903+00:00
[2024-12-29T16:34:14.015+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=131) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-29T16:34:14.020+0000] {standard_task_runner.py:72} INFO - Started process 133 to run task
[2024-12-29T16:34:14.019+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'gold_finance_data_pipeline', 'pyspark_task', 'manual__2024-12-29T16:34:08.220903+00:00', '--job-id', '198', '--raw', '--subdir', 'DAGS_FOLDER/Gold_finance_etl_dags.py', '--cfg-path', '/tmp/tmpdcnde0_e']
[2024-12-29T16:34:14.023+0000] {standard_task_runner.py:105} INFO - Job 198: Subtask pyspark_task
[2024-12-29T16:34:14.107+0000] {task_command.py:467} INFO - Running <TaskInstance: gold_finance_data_pipeline.pyspark_task manual__2024-12-29T16:34:08.220903+00:00 [running]> on host 77bcd09e75b5
[2024-12-29T16:34:14.345+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gold_finance_data_pipeline' AIRFLOW_CTX_TASK_ID='pyspark_task' AIRFLOW_CTX_EXECUTION_DATE='2024-12-29T16:34:08.220903+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-29T16:34:08.220903+00:00'
[2024-12-29T16:34:14.348+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-29T16:34:26.664+0000] {logging_mixin.py:190} INFO - An error occurred: [PATH_NOT_FOUND] Path does not exist: file:/app/dags/tmp/raw/alpha_vantage.csv.
[2024-12-29T16:34:26.665+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-29T16:34:26.697+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-29T16:34:26.697+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=gold_finance_data_pipeline, task_id=pyspark_task, run_id=manual__2024-12-29T16:34:08.220903+00:00, execution_date=20241229T163408, start_date=20241229T163413, end_date=20241229T163426
[2024-12-29T16:34:26.764+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-29T16:34:26.806+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-29T16:34:26.810+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
